### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by jschueller**
This should go into 1.18
- CI: Parallelize sphinx build
Locally:
j1=71m
j2=57m
j4=28m
j8=19m
- #1911
- #1912
- Memoize marginals now share the same cache cc @regislebrun 
- #1915
- Dropped sobol algorithm warnings about bounds (happens too often even in low dim, unreadable in high dim) -> moved to doc
- #1918

### CODE REVIEW

File: python/src/SobolIndicesAlgorithmImplementation_doc.i.in [hunk 1]
```diff
@@ -187,6 +187,9 @@ Their confidence level can be adjusted with :meth:`setConfidenceLevel`.
 The default confidence level is stored in the :class:`~openturns.ResourceMap`
 and can be accessed with the `SobolIndicesAlgorithm-DefaultConfidenceLevel` key.
 
+Indices values can be slightly outside of [0,1] depending on the estimator convergence.
```
Comment (josephmure) [inline python/src/SobolIndicesAlgorithmImplementation_doc.i.in:190]:
```suggestion
Indices estimates can be slightly outside of [0,1] if the estimator has not converged.
```
To clear up any ambiguity.

File: python/src/SobolIndicesAlgorithmImplementation_doc.i.in [hunk 2]
```diff
@@ -187,6 +187,9 @@ Their confidence level can be adjusted with :meth:`setConfidenceLevel`.
...
+For the same reason some first order indices can be greater than the corresponding total order indices.
```
Comment (josephmure) [inline python/src/SobolIndicesAlgorithmImplementation_doc.i.in:191]:
```suggestion
For the same reason some first order indices estimates can be greater than the corresponding total order indices estimates.
```

**Review by josephmure — APPROVED** [review]
What I can understand looks good to me!

**Comment by josephmure** [comment]
About Splitter though, I do not understand why you want to remove public methods from the Python layer. How would you emulate what they do in Python?

**Comment by jschueller** [comment]
> About Splitter though, I do not understand why you want to remove public methods from the Python layer. How would you emulate what they do in Python?

these are only needed for c++

**Pull request merged**